src/__tests__/helpers/appsync.test.ts

```diff
@@ -13,7 +13,7 @@ describe('appSyncMappingTemplate', () => {
             foo: 'bar',
           },
         },
-      }),
+      }) as object,
```

Comment:
Note: these are our internal tests. We need to cast as `object` otherwise TypeScript complains. 

Here we actually want to test the output of the helper function.